app/controllers/spree/admin/analysis_controller.rb

```diff
@@ -47,6 +48,10 @@ def download
       end
 
       private
+        def set_current_store
+          params[:store] = current_store
```

Comment:
why it is needed? I belive spree has method to get current store for all admin panel calls

Reply:
the **set_current_store**  method is defined in Spree::Admin::ResourceController but in our gems analysis controller we are inheriting from Spree::Admin::BaseController, so we cannot  access current store from the defined method in spree.